### PR TITLE
Update lib.py

### LINKED ISSUE #15: lib.info should be lib.album or lib.album_info
"info" is simply too generic. This information should line up with ``album`` in our code reference https://github.com/siznax/moo/wiki/%F0%9F%8E%B7-Reference

### PR DESCRIPTION
**Pull request by khafrewatkins**
Here are the changes you requested.  Please let me know any corrections or additional alterations need to be made.  I commented out the affected code blocks and adjustments. I know github tracks changes but just to make it more visible.

### CODE REVIEW

**Comment by siznax** [comment]
Thanks for the PR @khafrewatkins! :partying_face:

You commented out ``lib.albums()`` (note the "s" on the end—sorry, that's confusing), but we want to leave that function the way it is—so you can _comment_ it back in.

You commented out ``lib.album()`` which is what we want. That function is not being used, and is too close to ``lib.albums()``. So, go ahead and _delete_ that function in your PR code. Don't worry, none of this code will be merged until we're ready.

You changed ``lib.info()`` to ``lib.album()`` which is what I asked for. However, now I see that having ``albums()`` and ``album()`` is confusing. Let's change ``lib.info()``  to ``lib.album_info()`` instead.

If you're still up for it, just make those changes on your PR branch, commit, and push them. They should show up here for us to continue working with.

**Review by siznax — CHANGES_REQUESTED** [review]
Thanks for the PR @khafrewatkins! :partying_face:

I put my comments in a [PR Review](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews)

File: Moo/play/lib.py [hunk 1]
```diff
@@ -51,46 +51,46 @@
...
+#             if path not in dupes:
+#                 dupes[path] = list()
+#             dupes[path].append(path)
+#         else:
+#             out[path] = pruned
+
+#     if dupes:
+#         raise ValueError('Found duplicates: {}'.format(dupes))
+
+#     return out
```
Comment (siznax) [inline Moo/play/lib.py]: You commented out lib.albums() (note the "s" on the end—sorry, that's confusing), but we want to leave that function the way it is—so you can comment it back in.

File: Moo/play/lib.py [hunk 2]
```diff
@@ -108,13 +108,13 @@ def alpha(albums, mtag='artist'):
...
-    for path in index:
-        if str(path).endswith(key):
-            return str(path)
+# def album(key, index):
+#     ''' Removing/Erasing
+#     returns album from index matching key
+#     '''
+#     for path in index:
+#         if str(path).endswith(key):
+#             return str(path)
```
Comment (siznax) [inline Moo/play/lib.py]: You commented out lib.album() which is what we want. That function is not being used, and is too close to lib.albums(). So, go ahead and delete that function. Don't worry, none of this code will be merged until we're ready.

File: Moo/play/lib.py [hunk 3]
```diff
@@ -354,8 +354,9 @@ def index(config, sort=None):
     return sorted(albums, key=sort_key, reverse=True)
 
 
-def info(track, metadata):
+def album(track, metadata):
     '''
+    Changed def info to def album()
```
Comment (siznax) [inline Moo/play/lib.py]: You changed lib.info() to lib.album() which is what I asked for. However, now I see that having albums() and album() is confusing. Let's change lib.info() to lib.album_info() instead.

**Review by siznax — CHANGES_REQUESTED** [review]

**Comment by khafrewatkins** [comment]
Oh okay got it!  I will make those changes.

File: Moo/play/lib.py [hunk 4]
```diff
@@ -51,46 +51,46 @@
...
-#             out[path] = pruned
-
-#     if dupes:
-#         raise ValueError('Found duplicates: {}'.format(dupes))
-
-#     return out
+def albums(base, index):
+    ''' Removing/Erasing
+    returns dict of minimal album metadata
+    '''
```
Comment (siznax) [inline Moo/play/lib.py]: This is  a [docstring](https://www.python.org/dev/peps/pep-0257/) which should not be used for code comments. Let's remove "Removing/Erasing" here.

File: Moo/play/lib.py [hunk 5]
```diff
@@ -354,9 +347,9 @@ def index(config, sort=None):
     return sorted(albums, key=sort_key, reverse=True)
 
 
-def album(track, metadata):
+def album_info(track, metadata):
```
Comment (siznax) [inline Moo/play/lib.py:351]: Perfect.

File: Moo/play/lib.py [hunk 6]
```diff
@@ -354,9 +347,9 @@ def index(config, sort=None):
...
     '''
-    Changed def info to def album()
+    Changed def info to def album_info()
```
Comment (siznax) [inline Moo/play/lib.py]: Please remove "Changed def info to def album_info()" from this docstring.

**Review by siznax — CHANGES_REQUESTED** [review]
Looking good, but we need to keep code change comments out of docstrings. Use the ``-m <msg>`` option for that when you commit. See our project's commit history for examples: https://github.com/siznax/moo/commits/main

**Comment by khafrewatkins** [comment]
Ahhh okay.  I will do that know thanks for guiding a newb through this.

**Comment by siznax** [comment]
Thanks for your help @khafrewatkins. It looks like we lost the entire docstring from ``lib.albums()``. We'll have to restore that. I can't merge this pull request at this point, because it will break the main branch. Once I start developing unit tests, that will be apparent in GitHub.

Remember, there is no such thing as a "newbie" because no one can know everything. We all have something to learn _from each other_.

Let me know if you want to continue working on this. We need to satisfy all of #15 in order to merge.

We're _almost_ there 😃

**Comment by khafrewatkins** [comment]
Yes I will do whatever needs to be done!

File: Moo/play/lib.py [hunk 7]
```diff
@@ -52,7 +52,7 @@
 
 
 def albums(base, index):
-    '''
+    ''' Removing/Erasing
```
Comment (siznax) [inline Moo/play/lib.py]: Please remove "Removing/Erasing" but leave the docstring.

**Review by siznax — CHANGES_REQUESTED** [review]

**Review by siznax — APPROVED** [review]

**Comment by siznax** [comment]
Thanks for your help @khafrewatkins!

**Pull request merged**